browser-test/test.browser.ts

```diff
@@ -14,15 +14,9 @@
 import assert from 'assert';
 import {describe, it} from 'mocha';
 import {request} from '../src/index';
-import {PassThrough} from 'stream';
-import * as stream from 'stream';
 import * as uuid from 'uuid';
 const port = 7172; // should match the port defined in `webserver.ts`
 
-function isReadableStream(obj: stream.Readable | string) {
```

Comment:
was this just not used?

Reply:
Yes, apparently! TypeScript raised a warning and there were no errors in tests after I deleted it so I assumed it was okay.